.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*  @anzboi @anzdaddy @camh-anz @Joshcarp @juliaogris @linuxmonk @rokane
+*  @anzboi @anzdaddy @camh-anz @Joshcarp @juliaogris @linuxmonk @rokane @pentaphobe
```

Comment:
This change shouldn't be in your PR.

Reply:
Yeah, but once branch refreshed, they are gone.